src/assisted_test_infra/test_infra/utils/utils.py

```diff
@@ -580,9 +580,9 @@ def get_assisted_controller_status(kubeconfig):
 
 
 def download_iso(image_url, image_path):
-    with requests.get(image_url, stream=True, verify=False) as image, open(image_path, "wb") as out:
-        for chunk in image.iter_content(chunk_size=1024):
-            out.write(chunk)
+    run_command(
+        f"curl --location {image_url} --retry 10 --retry-connrefused -o {image_path} --continue-at -"
+    )
```

Comment:
```suggestion
@retry(...)
def download_iso(image_url, image_path):
    with requests.get(image_url, stream=True, verify=False) as image, open(image_path, "wb") as out:
        for chunk in image.iter_content(chunk_size=1024):
            out.write(chunk)
    )
```

Reply:
I considered doing it this way but we lose "resume" functionality offered by curl.